src/julia.h

```diff
@@ -2090,6 +2090,10 @@ typedef struct {
 
     // Cache access. Default: jl_rettype_inferred.
     jl_codeinstance_lookup_t lookup;
+
+    // If non-null, rewrite all generic calls to call
+    // generic_context(f, args...) instead of f(args...).
+    jl_value_t *generic_context;
```

Comment:
Needs a default value in `jl_default_cgparams`?

Reply:
Yes. Also I realize the comment is outdated, since I settled on `nothing` rather than null to indicate absence.